### PR TITLE
Ft/fix hero section

### PR DESCRIPTION
**Pull request by christelle-gihozo**
redo the hero section to be responsive on all breakpoints basing on the review we received

### CODE REVIEW

File: src/sections/hero.html [hunk 1]
```diff
@@ -1,68 +1,19 @@
...
->
-  <div
-    class="absolute right-11 top-[15.5rem] min-h-s-2 max-w-s-2 md:right-[6.8rem] md:top-[19rem] md:max-h-md-2 md:max-w-md-2 lg:right-[34rem] lg:top-[8rem] lg:max-h-l-2 lg:max-w-l-2"
-  >
-    <img
-      class="h-full w-full object-cover"
-      src="../../img/pink.png"
-      alt="pink vector"
-    />
+<div class="h-[64.208rem] lg:mx-auto relative -mt-96 md:mt-8 lg:mt-20">
```
Comment (kevin-Nk) [inline src/sections/hero.html]: classes are not well arranged

File: src/sections/hero.html [hunk 2]
```diff
@@ -1,68 +1,19 @@
...
+  <div class="w-full h-screen md:min-h-screen lg:mt-0">
+    <img src="../../img/left-green.svg" alt="green shape" class="z-[5] lg:z-[4] absolute bottom-[17%] md:bottom-[29%] left-[3%] lg:top-[14.341%] lg:left-[20.54%] w-[56%] lg:w-[34.61%]"/>
```
Comment (kevin-Nk) [inline src/sections/hero.html]: same here

File: src/sections/hero.html [hunk 3]
```diff
@@ -1,68 +1,19 @@
...
+    <img src="../../img/red.svg"  alt="red shape" class="z-[4] lg:z-[3] absolute bottom-[4.7%] md:bottom-[8.2%] right-[5%] lg:left-[13.04%] lg:top-[43%] w-[73.5%] lg:w-[47.66%]"/>
+    <img src="../../img/blue.svg" alt="blue shape" class="z-[3] lg:z-[5] absolute bottom-[11%] md:bottom-[19%] lg:top-[6.7%] w-[70%] lg:w-[46.4993395%] right-[5%] lg:left-[41.22%]"/>
+    <img src="../../img/yellow.svg" alt="yellow shape" class="z-[2] absolute left-0 bottom-[7.5%] md:bottom-[12%] lg:top-0 lg:left-0 w-[76%] lg:w-[47.95%]"/>
+    <img src="../../img/right-green.svg" alt="Green shape" class="z-[-10] absolute bottom-0 lg:right-0 lg:top-[24.42%] right-0 w-[42%] lg:w-[26.82%] "/>
+    <img src="../../img/pink.svg" alt="Pink shape" class="absolute bottom-[23.3%] md:bottom-[43%] right-[9%] w-[30%] lg:w-[15.59%] lg:left-[39.56%] lg:top-[2.27%]"/>
```
Comment (kevin-Nk) [inline src/sections/hero.html]: Here it's good

File: src/sections/hero.html [hunk 4]
```diff
@@ -1,68 +1,19 @@
...
+    <div id="hero-text" class="absolute z-30 inset-0 lg:flex justify-center lg:items-center top-[69%] md:top-[39%] lg:-top-[20%]">
```
Comment (kevin-Nk) [inline src/sections/hero.html]: Same here check the classes.

File: src/sections/hero.html [hunk 5]
```diff
@@ -1,68 +1,19 @@
...
+          <p class="w-auto text-center font-outfit text-2xl lg:text-5xl md:text-4xl text-primaryColor">
```
Comment (kevin-Nk) [inline src/sections/hero.html]: same here!

**Review by kevin-Nk — CHANGES_REQUESTED** [review]
I left some comments check them out

File: src/sections/hero.html (hunk 1 above)
Comment (christelle-gihozo) [reply src/sections/hero.html]: I had arranged them now, thank you

File: src/sections/hero.html (hunk 2 above)
Comment (christelle-gihozo) [reply src/sections/hero.html]: I have arranged them, thank you.

File: src/sections/hero.html (hunk 3 above)
Comment (christelle-gihozo) [reply src/sections/hero.html]: thank you

File: src/sections/hero.html (hunk 4 above)
Comment (christelle-gihozo) [reply src/sections/hero.html]: I have arranged them

File: src/sections/hero.html (hunk 5 above)
Comment (christelle-gihozo) [reply src/sections/hero.html]: I have arranged them

**Review by kevin-Nk — APPROVED** [review]
this looks nice now

**Comment by kevin-Nk** [comment]
Hope it's okay now

**Review by kevin-Nk — APPROVED** [review]

**Pull request merged**